homeassistant/components/switch/homekit_controller.py

```diff
@@ -62,3 +70,20 @@ def turn_off(self, **kwargs):
                             'iid': self._chars['on'],
                             'value': False}]
         self.put_characteristics(characteristics)
+
+    @property
+    def state_attributes(self):
+        """Return the optional state attributes."""
+        data = {}
+
+        for prop, attr in PROP_TO_ATTR.items():
```

Comment:
If it's only one attribute we don't need a dict or use `getattr`. Keep it simple for now.

Reply:
ok